SoftLayer/CLI/block/access/revoke.py

```diff
@@ -0,0 +1,38 @@
+"""Create a block storage snapshot."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+import SoftLayer
+from SoftLayer.CLI import environment
+
+
+@click.command()
+@click.argument('volume_id')
+@click.option('--hardware_id', '-h', multiple=True,
```

Comment:
Use dashes in favor of underscores for CLI options. `--hardware-id` over `--hardware_id`.